src/java.xml/share/conf/jaxp-compat.properties

```diff
@@ -0,0 +1,116 @@
+################################################################################
+#           JAXP Compatibility Configuration File
+#
+# ** Warning: unsafe configuration, use with security assessment **
+#
+# jaxp-compat.properties is a JAXP configuration file with compatible settings
+# prior to a strict configuration such as jaxp-strict.properties. In this
+# configuration, properties that have more restrictive settings as in the
+# strict configuration (jaxp-strict.properties) are reversed back to their
+# defaults. In particular:
```

Comment:
I think the above needs some more word smithing  as we have not articulated what a strict configuration is or how this differs from jaxp.properties